src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -43,6 +43,9 @@ set(
   data_loader.cc
   concurrency_manager.cc
   request_rate_manager.cc
+  worker.cc
```

Comment:
Can we rename this file/class to LoadWorker?

Reply:
Yup! That was one of the questions I had. What to name these classes.  I was considering adding 'thread' somewhere in the names. If you're ok with the other names I'm ok with LoadWorker or WorkerBase.

Reply:
done